spec/features/rental_creation_spec.rb

```diff
@@ -46,7 +46,7 @@
           fill_in('rental_end_time', with: Date.today.strftime('%Y-%m-%d'))
 
           expect(page).to have_button('rentalSubmit', disabled: true)
-          check('TOC')
+          # check('TOC')
```

Comment:
Why'd you comment this out?

Reply:
Trying to see if I can get something working using the new logic @tomecho wrote that it [removes the checkbox if a person's department is Parking](https://github.com/umts/probable-engine/blob/master/app/views/rentals/_form.html.erb#L71).